lib/qa/authorities/crossref/generic_authority.rb

```diff
@@ -0,0 +1,59 @@
+module Qa::Authorities
+  class Crossref::GenericAuthority < Base
+    include WebServiceBase
+    class_attribute :label, :identifier
+    attr_reader :subauthority
+
+    def initialize(subauthority)
+      @subauthority = subauthority
+    end
+
+    self.label = lambda do |item|
+      [item['name'],
+       item['alt-names'].blank? ? nil : "(#{item['alt-names'].join(', ')})",
+       item['location']].compact.join(', ')
+    end
```

Comment:
This looks clever, but I'm confused by it. I don't use lambdas very much. It looks like returning an array with a single hash in it. Perhaps some YARD documentation here to explain that. Also, why this way and not just another method call?